pandas/core/resample.py

```diff
@@ -549,6 +550,14 @@ def var(self, ddof=1, *args, **kwargs):
         nv.validate_resampler_func('var', args, kwargs)
         return self._downsample('var', ddof=ddof)
 
+    @Appender(GroupBy.size.__doc__)
+    def size(self):
+        # It 'seems' special and needs extra handling. GH14962
```

Comment:
better comment here. say its a special case as higher level does returns a copy of 0-len objects.